regulations/static/regulations/css/less/module/drawer-content.less

```diff
@@ -119,7 +119,7 @@ This contains all of the styles specific to the history
     .effective-label {
         padding: 0 15px;
         .font-regular;
-        color: @dark_gray;
+        color: #666666;
```

Comment:
I'd be more in favor of darkening/lightening an existing color than adding a new one.